docs/usage.rst

```diff
@@ -3,35 +3,66 @@ Usage
+parameters and model input directly from the command line without
+coding themselves. Please see :ref:`tool` for its usage documentation.
 
 
 .. _parameters:
 
 Parameters
 ----------
 
-**Parameter** here refers to any named input or output variable of a model. A parameter can be either scalar (i.e. a single number) or a timeseries and has a unique name in a hierarchy of arbitrary depth. That means every parameter either is a root parameter without a parent parameter ("level 0") or belongs to a parent parameter.
+**Parameter** here refers to any named input or output variable of a
+model. A parameter can be either scalar (i.e. a single number) or a
+timeseries and has a unique name in a hierarchy of arbitrary depth.
+That means every parameter either is a root parameter without a parent
+parameter ("level 0") or belongs to a parent parameter.
 
-For example, the parameter for industrial carbon emissions belongs to the parameter for carbon emissions, which in turn belongs to the root parameter emission. Thus, it is identified by
+For example, the parameter for industrial carbon emissions belongs to
+the parameter for carbon emissions, which in turn belongs to the root
+parameter emission. Thus, it is identified by
 
     ``Emissions`` -> ``CO2`` -> ``Industrial``.
 
-In the low-level API parameters are expected to be identified by tuples of strings describing their position in the hierarchy, i.e. in this example ``("Emissions", "CO2", "Industrial")``. The high-level API also allows for giving it as a string with the hierarchical levels separated by ``|``, e.g. ``"Emissions|CO2|Industrial"``.
+In the low-level API parameters are expected to be identified by
+tuples of strings describing their position in the hierarchy, i.e. in
+this example ``("Emissions", "CO2", "Industrial")``. The high-level
+API also allows for giving it as a string with the hierarchical levels
+separated by ``|``, e.g. ``"Emissions|CO2|Industrial"``.
 
 See :ref:`parameter-hierarchy` for the standard parameters in OpenSCM.
 
 
 Time frames
 -----------
 
-Timeseries parameters are always given with a corresponding time frame, which consists of a time point and a period length. The time point gives the start of the timeseries; the period length gives the length of the period between consecutive values in the timeseries. Each parameter value is assumed to be the **average value** for its corresponding period. This implies that additive values such as emissions need to be given as a rate, e.g. ``tC/a`` rather than ``tC``.
+Timeseries parameters are always given with a corresponding time
+frame, which consists of a time point and a period length. The time
+point gives the start of the timeseries; the period length gives the
+length of the period between consecutive values in the timeseries.
+Each parameter value is assumed to be the **average value** for its
+corresponding period. This implies that additive values such as
```

Comment:
```suggestion
corresponding period. This implies that flux variables such as
```

Reply:
Hmm, sounds non-standard to me. We should find a compromise on how to call these... (this PR is only about reformatting and including additional modules in the docs, though)

Reply:
maybe flux is too closely related to electromagnetism, how about

```suggestion
corresponding period. This implies that values which define the flow/transport of a quantity (such as emissions, flow of a gas, and radiative forcing, flow of energy) must be given as a rate
```